### PR TITLE
seal-level rise fixes

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

**Comment by mhines-usgs** [comment]
This is something Jordan Read noticed.

File: coastal-hazards-portal/src/main/webapp/META-INF/context.xml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<Context antiJARLocking="true" path="/coastal-hazards-portal"/>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/META-INF/context.xml]: Darn those Windows line endings.

**Comment by mhines-usgs** [comment]
Sad face @jiwalker-usgs

**Comment by isuftin** [comment]
@jiwalker-usgs @mhines-usgs This has been outstanding since August. :hankey: or get off the :toilet: on this one?

**Comment by jread-usgs** [comment]
this is sad

**Comment by isuftin** [comment]
Looks legit enough :D

**Comment by mhines-usgs** [comment]
I didn't want to cut a release for just that :)

**Pull request merged**